Tests/src/PublicAPITests/ECommerceSystemTests.cpp

```diff
@@ -136,4 +136,45 @@ CSP_PUBLIC_TEST(CSPEngine, ECommerceSystemTests, GetProductInformationTest)
 
 	LogOut(UserSystem);
 }
-#endif
\ No newline at end of file
+#endif
+
+#if RUN_ECOMMERCE_TESTS || RUN_ECOMMERCE_GET_CHECKOUT_INFORMATION_TEST
+CSP_PUBLIC_TEST(CSPEngine, ECommerceSystemTests, GetCheckoutInformationTest)
+{
+	SetRandSeed();
+
+	auto& SystemsManager  = csp::systems::SystemsManager::Get();
+	auto* UserSystem	  = SystemsManager.GetUserSystem();
+	auto* ECommerceSystem = SystemsManager.GetECommerceSystem();
+
+	csp::common::String UserId;
+	LogIn(UserSystem, UserId);
+	auto Details						   = GetShopifyDetails();
+	const csp::common::String FalseSpaceId = "abcdefghijk1234567891011";
+	const csp::common::String FalseCartId  = "B1-1234567891011121314151617e8e21er";
+
+	// The additional info inside of this mask need to be added to the ShopifyCred.txt file on a new line as: "Key Value"
```

Comment:
Can you add a little more details about what this comment is actually saying. I don't really get it.

Reply:
sorry typo, should be sorted